src/card.css

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
```

Comment:
redundant, block element placed one below others by default

Reply:
my code broke without it